resources/js/Components/Galleries/GalleryPage/GalleryNftsNft.tsx

```diff
@@ -80,14 +79,13 @@ export const GalleryNftsNft = ({ nft, isSelected, onClick }: Properties): JSX.El
                             />
                         </div>
                     )}
-                    <Link
-                        external
-                        href={nft.collectionWebsite}
+                    <div
```

Comment:
if we make a `div` clickable, it's best to add `role="button"` for accessibility. Although, why isn't this still a `Link` where you remove the `external` attribute as it's no longer an external link?